syfertext/doc.py

```diff
@@ -123,35 +123,30 @@ def __iter__(self):
             # Yield a Token object
             yield self[i]
 
-    def getEncryptedVector(self, *workers, crypto_provider=None, requires_grad=True):
+    def get_vector(self, *workers, crypto_provider=None, requires_grad=True):
         """
-           Create one big vector composed of the concatenated Token vectors included in the
-           Doc. The returned vector is SMPC-encrypted.
-
-           TODO: This method should probably be removed. It served for a prototype test,
-                 but concatenating all token vectors of the Doc into one big vector
-                 might not be really useful for practical usecases.
+           Get the mean of the vectors of each Token in this documents.
+           Note: The token vectors are SMPC-encrypted.
         """
 
         assert (
             len(workers) > 1
         ), "You need at least two workers in order to encrypt the vector with SMPC"
 
         # Accumulate the vectors here
-        vectors = []
+        vectors = None
 
         for token in self:
 
             # Get the encypted vector of the token
-            vectors.append(
-                token.getEncryptedVector(
-                    *workers,
-                    crypto_provider=crypto_provider,
-                    requires_grad=requires_grad
-                )
+            vector = token.getEncryptedVector(
+                *workers,
```

Comment:
Could you please rename it  to `token.get_encrypte_vector` which following the PEP 8 style guide

https://www.python.org/dev/peps/pep-0008/#id45

(I hadn't pay attention to that when I chose the name)

Reply:
OK i renamed everything to `get_encrypte_vector `